# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/base/crashtests/crashtests.list
# Commit: 3aa74b745aa5
# Full Hash: 3aa74b745aa5ae1f1b460452f223ad0b9633fec0
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2025-05-13 09:26:34
# Description:
#   Bug 1965664 - Fix some crashes in ContentSubtreeIterator r=jjaschke,dom-core
#   
#   by skipping the shadow hosts that ShadowDOM selection doesn't
#   support at the moment.
#   
# ==============================================================================

diff -r ecccb8942de4 -r 3aa74b745aa5 dom/base/crashtests/crashtests.list
--- a/dom/base/crashtests/crashtests.list	Mon May 12 20:34:06 2025 +0000
+++ b/dom/base/crashtests/crashtests.list	Mon May 12 20:40:53 2025 +0000
@@ -283,3 +283,4 @@
 load 1907464.html
 asserts(0-1) load 1907228.html # see bug 1908485 for assert
 load 1922153.html
+load 1965664.html
